app/controllers/appropriate_bodies/participants_controller.rb

```diff
@@ -2,6 +2,16 @@
 
 module AppropriateBodies
   class ParticipantsController < BaseController
-    def index; end
+    def index
+      appropriate_body
+    end
+
+  private
+
+    def appropriate_body
+      @appropriate_body ||= current_user.appropriate_bodies.find(params[:appropriate_body_id])
+    end
+
+    helper_method :appropriate_body
```

Comment:
how come a change in approach here? I personally prefer this method over `before_action` as they are lazy

Reply:
@asmega 
I guess this is more to stay consistent with what we do in other controllers, we don't use `before_action` much.